initializers/prefixes.yml

```diff
@@ -0,0 +1,12 @@
+#- description: prefix1
+#  prefix: 10.1.1.0/24
+#  site: AMS 1
+#  status: 1
+#  tenant: tenant1
+#  vlan: vlan1
+#- description: prefix2
+#  prefix: 10.1.2.0/24
+#  site: AMS 2
+#  status: 1
```

Comment:
Would it be possible to use the actual word here, i.e. `Active`, `Container`, `Reserved`, `Deprecated`?

See https://github.com/netbox-community/netbox-docker/blob/master/startup_scripts/110_devices.py#L52-L55 for how this was made for the `face` property in `devices.yml`.